src/main/scala/scalatutorial/sections/ImperativeProgramming.scala

```diff
@@ -115,9 +115,11 @@ object ImperativeProgramming extends ScalaTutorialSection {
    *       balance
    *     }
    *     def withdraw(amount: Int): Int =
-   *       if (0 < amount && amount <= balance) {
+   *       if (0 <= amount && amount <= balance) {
    *         balance = balance - amount
    *         balance
+   *       } else if (amount < 0) {
+   *         throw new Error("invalid withdraw amount")
```

Comment:
since, to me, it doesn't make sense to withdraw 0.
```suggestion
   *       if (0 < amount && amount <= balance) {
   *         balance = balance - amount
   *         balance
   *       } else if (amount <= 0) {
   *         throw new Error("invalid withdrawal amount")
```

Reply:
👍